020-Connect/010-Postgres/001-connect.mdx

```diff
@@ -0,0 +1,194 @@
+| `ALTER INDEX`           | N             | N              | Y                 |
+| `DROP INDEX`            | N             | N              | Y                 |
+| `ALTER TRIGGER`         | N             | N              | Y                 |
+| `CREATE TRIGGER`        | N             | N              | Y                 |
+| `DROP TRIGGER`          | N             | N              | Y                 |
+| `ALTER EVENTTRIGGER`    | N             | N              | Y                 |
+| `CREATE EVENTTRIGGER`   | N             | N              | Y                 |
+| `DROP EVENTTRIGGER`     | N             | N              | Y                 |
+| `ALTER SEQUENCE`        | N             | N              | Y                 |
+| `CREATE SEQUENCE`       | N             | N              | Y                 |
+| `DROP SEQUENCE`         | N             | N              | Y                 |
+| `CALL`                  | N             | N              | Y                 |
+| `ALTER STATISTICS`      | N             | N              | Y                 |
+| `CREATE STATISTICS`     | N             | N              | Y                 |
+| `DROP STATISTICS`       | N             | N              | Y                 |
+| `ALTER RULE`            | N             | N              | Y                 |
+| `CREATE RULE`           | N             | N              | Y                 |
+| `DROP RULE`             | N             | N              | Y                 |
+| `ALTER TYPE`            | N             | N              | Y                 |
+| `CREATE TYPE`           | N             | N              | Y                 |
+| `DROP TYPE`             | N             | N              | Y                 |
+| `ALTER DOMAIN`          | N             | N              | Y                 |
+| `CREATE DOMAIN`         | N             | N              | Y                 |
+| `DROP DOMAIN`           | N             | N              | Y                 |
+| `ALTER FUNCTION`        | N             | N              | Y                 |
+| `CREATE FUNCTION`       | N             | N              | Y                 |
+| `DROP FUNCTION`         | N             | N              | Y                 |
+| `EXPLAIN`               | N             | N              | Y                 |
+| `REINDEX`               | N             | N              | Y                 |
+| `ALTER OPERATOR CLASS`  | N             | N              | Y                 |
+| `CREATE OPERATOR CLASS` | N             | N              | Y                 |
+| `DROP OPERATOR CLASS`   | N             | N              | Y                 |
+| `CREATE EXTENSION`      | N             | N              | Y                 |
+| `DROP EXTENSION`        | N             | N              | Y                 |
+| `ALTER EXTENSION`       | N             | N              | Y                 |
+| Temporary tables        | N             | N              | Y                 |
```

Comment:
Maybe `CREATE TEMPORARY TABLE` is more consistent than "Temporary tables". WDYT?

Reply:
resolved via https://github.com/xataio/mdx-docs/pull/281/commits/fc845dc8671ee7763ea431d68c2ccc45d690f4e0